.gitignore

```diff
@@ -1,3 +1,5 @@
+index.yaml
+
```

Comment:
please remove this from .gitignore since it is something you only use locally for your personal experimentation.